src/main/features/claim/views/details-summary.njk

```diff
@@ -70,9 +79,9 @@
         {% else %}
           {% set defendantFullName = defendant.defendantDetails.fullName %}
         {% endif %}
-        {{ row('Name', defendantFullName) }}
+        {{ row('Name', defendantFullName, changeDefendantLink) }}
```

Comment:
yes, that's what it is doing. The variable `changeDefendantLink` is being populated above in else block at line 70 based on the defendant count.